test/e2e/dra/dra.go

```diff
@@ -907,8 +907,15 @@ var _ = framework.SIGDescribe("node")("DRA", feature.DynamicResourceAllocation,
 				for _, nodeName := range nodes.NodeNames {
 					expectedObjects = append(expectedObjects,
 						gstruct.MatchAllFields(gstruct.Fields{
-							"TypeMeta":   gstruct.Ignore(),
-							"ObjectMeta": gstruct.Ignore(), // TODO (https://github.com/kubernetes/kubernetes/issues/123692): validate ownerref
+							"TypeMeta": gstruct.Ignore(),
+							"ObjectMeta": gstruct.MatchFields(gstruct.IgnoreExtras, gstruct.Fields{
+								"OwnerReferences": gstruct.MatchElements(func(element interface{}) string { return element.(metav1.OwnerReference).Kind }, gstruct.IgnoreExtras, gstruct.Elements{
+									"OwnerReference": gstruct.MatchFields(gstruct.IgnoreExtras, gstruct.Fields{
+										"kind": gomega.Equal("Node"),
+										"name": gomega.Equal(nodeName),
+									}),
```

Comment:
@pohly I failed to figure out how to correctly do this. Can you help?
Is it possible to check that the length of the `OwnerReferences` are always 1 using gstruct API?

Reply:
Thank you! applied.